PowerApps/Invoice Automation/readme.md

```diff
@@ -0,0 +1,106 @@
+<h1>Invoice Automation using the Power Platform</h1>
+
+Invoice processing is something that every company does, but many are still doing it manually and is very labor intensive.  Not anymore!  This starter kit is intended to use the new "Process and save information from invoices" flow in Power Automate to automatically capture invoice information, save it to a SharePoint document library.  We’ll then use Cognitive Services to extract the invoice line-item information and save it to a SharePoint List.  Finally, Power Apps will allow you to review and complete the processing so it can be integrated with the accounting LOB application.
+
+For an overview of the solution please refer to this [blog article]("https://powerusers.microsoft.com/t5/Power-Automate-Community-Blog/Invoice-Automation-using-the-Power-Platform/ba-p/875628) for details.
```

Comment:
Any chance you can add the overview here as well? You can keep the blog link in but this repo is intended to be the single source of all info, so the user should be able to find all info in this README

Reply:
Yes...I'll try to have that in this week.